scijava/scijava-ops/pom.xml

```diff
@@ -113,17 +113,6 @@
 				<groupId>com.alexecollins.maven.plugin</groupId>
 				<artifactId>script-maven-plugin</artifactId>
 			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
Ah cool, ignore my previous comment about this, then!

Reply:
Ignoring!